lib/address_encrypted.ex

```diff
@@ -20,5 +20,4 @@ defmodule Fields.AddressEncrypted do
 
   def load(value), do: Encrypted.load(value)
 
-  def load(value, key_id), do: Encrypted.load(value, key_id)
```

Comment:
I've removed the `load\2` function as the key_id is now extracted from the value itself